prover/src/lib.rs

```diff
@@ -482,7 +477,7 @@ impl<C: SP1ProverComponents> SP1Prover<C> {
             let batched_compress_inputs =
                 compress_inputs.chunks(shard_batch_size).collect::<Vec<_>>();
             reduce_proofs = batched_compress_inputs
-                .into_iter()
+                .into_par_iter()
```

Comment:
this seems kind of free? Or is there a reason we chose to not do this concurrently? just lmk and I'll remove

Reply:
Ah no, good catch. That looks good :)